src/io.rs

```diff
@@ -14,6 +17,7 @@ use self::command_manager::CommandManager;
 use self::command_manager::KickReason;
 use self::filewatcher::FileWatcher;
 use self::g15::G15Parser;
+use self::g15::G15Player;
```

Comment:
Uhhhh, good question. I think I'm not too fussed as a lot of them are gonna be automatically added by the IDE. However it may be preferable to group them if there's a fair few.